prql-elixir/native/prql/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
This is ignored at the root, no need for this

Reply:
Removed